DEV-2/Program.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DEV_2
+{
+   public class Program
+    {
+        static void Main(string[] args)
+        {
+            string inputString = Console.ReadLine();
+            Program program = new Program();
+           
+            program.GetMaxNumberOfDifferentSymbolsConsecutive(inputString);
+            program.GetMaxNumberOfSameNumbersConsecutive(inputString);
+            program.GetMaxNumberOfSameLettersConsecutive(inputString);
+        }
+
```

Comment:
These three methods a very similar in behavior, think how to simplify and join their main functionality